### DISCUSSION TITLE
Log file volumes

### DISCUSSION THREAD

**Discussion by saloni-2404** [strimzi/strimzi-kafka-operator#4529] [Q&A]
Not able to find the log files volumes in kafka operator.
Need help for the same. Where the controller and servers logs we will be stored after deploying the kafka operator?
Not able to find the location in logs also.

**Comment by scholzj** [comment]
All logs are printed to standard output where they should be collected by your container runtime / Kubernetes and can be further processed with tools such as Fluentd, Logstash etc.

**Reply by saloni-2404** [reply]
So they are not stored anywhere? Is there any way to configure?

**Reply by scholzj** [reply]
They are stored short-term by your container runtime / Kubernetes and any long-term storage can be done using the further tools. This is more or less a common approach in Kubernetes
